src/universal/modules/meeting/components/MeetingUpdatesPrompt/MeetingUpdatesPrompt.js

```diff
@@ -2,30 +2,58 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import MeetingPrompt from 'universal/modules/meeting/components/MeetingPrompt/MeetingPrompt';
 import {createFragmentContainer} from 'react-relay';
+import {css} from 'aphrodite-local-styles/no-important';
+import withStyles from 'universal/styles/withStyles';
+import ui from 'universal/styles/ui';
 
 const MeetingUpdatesPrompt = (props) => {
   const {
     localPhaseItem,
-    team: {teamMembers}
+    team: {teamMembers},
+    meetingNumber,
+    // members,
+    styles
   } = props;
   const currentTeamMember = teamMembers[localPhaseItem - 1];
-  const heading = <span>{currentTeamMember.preferredName}, <i>what’s changed since our last meeting</i>?</span>;
+  // const heading = <span>{currentTeamMember.preferredName}, <i>what’s changed since our last meeting</i>?</span>;
+  const self = teamMembers.find((m) => m.isSelf);
+  const myTeamMemberId = self && self.id;
+  // const currentTeamMember = teamMembers[localPhaseItem - 1];
+  const isMyMeetingSection = myTeamMemberId === currentTeamMember.id;
+  const isFirstMeeting = meetingNumber === 1;
+  const handleAgendaControl = () => console.log('Focus on agenda input');
```

Comment:
@mattkrick mind taking a peek at this when you get a chance! A few things I did here: 1) added conditional question for first meeting and 2) added conditional message for you (the user) vs everybody else during rounds and 3) dropped in a handler to focus on the agenda input…needless to say, have things to clean up from conflicts etc.